pkg/v1/service/books/storage.go

```diff
@@ -0,0 +1,102 @@
+package books
+
+import (
+	"database/sql"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Transactor interface {
+	GetBookById(id uint) (Book, error)
```

Comment:
We could make this more generic; `GetRecordById`. This way, if we wanted to look up users, or another piece of data, we could easily reuse this package.

Reply:
Took a stab at it, let me know if that make sense.